proto/offers.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package offers;
+
+service Offers {
+    rpc PayOffer (PayOfferRequest) returns (PayOfferResponse);
+}
```

Comment:
Would be good to move this into an `lnkd` package/namespace.

Reply:
It should be since it's imported in lib.rs - unless I'm misunderstanding what you're saying?

Reply:
I mean to set `package lndk;` rather than `package offers;`. It would enable more clarity when using this alongside other interfaces.

e.g. lnd sets their top level namespace as `lnrrc`. So we end up with `lnrpc.Invoice` and `lndk.Offers` rather than `lnrpc.Invoice` and `offers.Offers`